src/services/penaltyDocuments.unit.js

```diff
@@ -0,0 +1,93 @@
+import expect from 'expect';
+import { doc } from 'serverless-dynamodb-client';
+import sinon from 'sinon';
+import PenaltyDocumentsService from './penaltyDocuments';
+import hashToken from '../utils/hash';
+import getUnixTime from '../utils/time';
+import mockPenaltyGroupsData from '../../mock-data/fake-penalty-groups.json';
+import mockPenaltiesData from '../../mock-data/fake-penalty-notice.json';
+
+
+describe('PenaltyDocuments service', () => {
+	/**
+	 * @type PenaltyDocumentsService
+	 */
+	let penaltyDocumentsService;
+	let callbackSpy;
+
+	beforeEach(() => {
+		penaltyDocumentsService = new PenaltyDocumentsService(doc, 'penaltyDocuments', '', '', '', '', '', 3, '');
+		callbackSpy = sinon.spy();
+		sinon.stub(doc, 'put')
+			.returns({
+				promise: () => Promise.resolve('put resolved'),
+			});
+	});
+	afterEach(() => {
+		// @ts-ignore
+		doc.get.restore();
+		// @ts-ignore
+		doc.put.restore();
+		callbackSpy.resetHistory();
+	});
+
+	describe('updateDocumentsUponPaymentDelete', () => {
+		it('calls back with OK status', async () => {
+			/**
+			 * Mock for db.get in updateDocumentUponPaymentDelete.
+			 * _tryUpdatePenaltyGroupToUnpaidStatus is stubbed so will not be called.
+			 */
+			sinon.stub(doc, 'get').returns({
+				promise: () => Promise.resolve({
+					Item: {
+						Value: {},
```

Comment:
Instead of defining the mock response in your test, there's a file with a bunch of mock payloads that you can return. `mock-data/fake-penalty-notice.json`. Similar to what you've done below for penalty groups.